index.ts

```diff
@@ -1,37 +1,37 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as eks from "@pulumi/eks";
 
+// Get the current stack name
+const currentStackName = pulumi.getStack();
+
 // Grab some configuration values
 const config = new pulumi.Config();
 const minClusterSize = config.getNumber("minClusterSize") || 3;
 const maxClusterSize = config.getNumber("maxClusterSize") || 6;
 const desiredClusterSize = config.getNumber("desiredClusterSize") || 3;
 const eksNodeInstanceType = config.get("eksNodeInstanceType") || "t3.medium";
-const infraOrgName = config.require("infraOrgName");
-const infraProjName = config.require("infraProjName");
-const infraStackName = config.require("infraStackName");
+const baseOrgName = config.get("baseOrgName") || "zephyr";
+const baseProjName = config.get("baseProjName") || "zephyr-infra";
+const baseStackName = config.get("baseStackName") || currentStackName;
 
 // Create a StackReference to get Kubeconfig from base stack
-const infraSr = new pulumi.StackReference(`${infraOrgName}/${infraProjName}/${infraStackName}`);
-const infraVpcId = infraSr.getOutput("vpcId");
-const infraPrivSubnetIds = infraSr.getOutput("privSubnetIds");
-const infraPubSubnetIds = infraSr.getOutput("pubSubnetIds");
+const baseSr = new pulumi.StackReference(`${baseOrgName}/${baseProjName}/${baseStackName}`);
+const baseVpcId = baseSr.getOutput("vpcId");
+const basePrivSubnetIds = baseSr.getOutput("privSubnetIds");
+const basePubSubnetIds = baseSr.getOutput("pubSubnetIds");
 
 // Create the EKS cluster
 const eksCluster = new eks.Cluster("eks-cluster", {
-    vpcId: infraVpcId,
-    publicSubnetIds: infraPubSubnetIds,
-    privateSubnetIds: infraPrivSubnetIds,
-    // Change configuration values to change any of the following settings
+    vpcId: baseVpcId,
+    publicSubnetIds: basePubSubnetIds,
+    privateSubnetIds: basePrivSubnetIds,
     instanceType: eksNodeInstanceType,
     desiredCapacity: desiredClusterSize,
     minSize: minClusterSize,
     maxSize: maxClusterSize,
     nodeAssociatePublicIpAddress: false,
-    // Uncomment the next two lines for a private cluster (VPN access required)
-    // endpointPrivateAccess: true,
-    // endpointPublicAccess: false
 });
 
 // Export some values for use elsewhere
 export const kubeconfig = eksCluster.kubeconfig;
+export const nodeSecurityGrp = eksCluster.nodeSecurityGroup.id;
```

Comment:
Nitpicking, but I would prefer to have `Group` instead of `Grp` in the output name.

Reply:
Changed in bef672c per your suggestion